clj/src/cljd/core.cljd

```diff
@@ -8246,4 +8246,51 @@
       (doseq [tap @tapset]
         (try
           (tap x)
-          (catch Exception ex))))))
\ No newline at end of file
+          (catch Exception ex))))))
+
+(defprotocol Inst
+  (inst-ms* [inst]))
```

Comment:
I think maybe more consistent names here would be `IInst` and `-inst-ms`, but I just copied the interface directly from ClojureScript.